pkg/liquidity-mining/contracts/BoostV2.vy

```diff
@@ -231,6 +231,16 @@ def boost(_to: address, _amount: uint256, _endtime: uint256, _from: address = ms
 
 @external
 def migrate(_token_id: uint256):
+    self._migrate(_token_id)
+
+@external
+def migrate_many(_token_ids: uint256[16]):
+    for i in range(16):
+        if (_token_ids[i] == 0) break
```

Comment:
The language of the future

Reply:
Also, this works because token ids can never be zero as they include the delegator address. See [line 609 of the preseeded delegation here](https://etherscan.io/address/0xB496FF44746A8693A060FafD984Da41B253f6790#code#L609).